### PR TITLE
Made cart page

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (sonishreyas) [inline .vscode/settings.json]: This file can be added in gitignore

**Review by sonishreyas — APPROVED** [review]
You can keep the colors as variables, rest looks good

File: .vscode/settings.json (hunk 1 above)
Comment (poojavishnoi) [reply .vscode/settings.json]: okay.

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            </button>
+            <button class="btn secondary_outline_btn" type="submit">
+              Move to wishlist
+            </button>
+          </div>
+        </div>
+
+      
+
+      <div class="price_details_container">
```
Comment (anandkumarcoder) [inline cart.html]: sugesstion: for listing price and value  ul tag can be used , easier and less css

**Review by anandkumarcoder — APPROVED** [review]
looks good for merging to me

File: cart.html (hunk 2 above)
Comment (poojavishnoi) [reply cart.html]: Okay will try that later.

**Pull request merged**